integration/cmd/historicalspeed-timezone-builder/flags.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+)
+
+var flags struct {
+	pbf       string
+	pbfSource string
+
+	inWaysMappingFile  string // LINK_PVID,TRAVEL_DIRECTION,U,M,T,W,R,F,S
+	outWaysMappingFile string // LINK_PVID,TRAVEL_DIRECTION,U,M,T,W,R,F,S,TIME_ZONE,DAYLIGHT_SAVING
+	withCSVHeader      bool   // whether dump with CSV header or not
+}
+
+const (
+	pbfSourceUniDB = "unidb"
```

Comment:
Nit: We could put this one to 
`integration/map/source_file_definition.go`

Reply:
Good idea. Maybe `integration/mapsource`?  Then use `mapsource.UniDB` or `mapsource.OSM`.